UITests/UITests.Tests.Shared/Tests.cs

```diff
@@ -113,5 +115,33 @@ public void SimpleLaunchTest()
 
             Verify.AreEqual("Clicked", textBlock.GetText());
         }
+
+        [TestMethod]
+        public void TestTextBoxMaskBinding_Property()
```

Comment:
I definitely want to figure out how we split these into multiple files as well, so we can have multiple test cases per test page. i.e. we can have a test page using the component with a setup or two, and then have multiple smaller test methods to run against it.

Though ideally, we figure out how to manage this without tearing down and re-opening the app each time?

@azchohfi as commented on the other PR, do you know how we'd structure this here or have suggestions?